apps/docs/src/medusa-forms/ControlledCheckbox.stories.tsx

```
@@ -0,0 +1,396 @@
+};
+
+// Required Field Validation Story
+const RequiredValidationForm = () => {
+  const form = useForm({
+    defaultValues: {
+      requiredField: false,
+    },
+    mode: 'onChange',
+  });
+
+  const onSubmit = (data: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)